GoogleForCreators/web-stories-wp
# Context menus: Move tooltip out of the MenuIcon component

## Context



## Summary
Fixes the issue where element in focus with keyboard does not show the tooltip.


## Relevant Technical Choices
Moves `Tooltip` out of the `MenuIcon` component to `MenuButton` component.
Since the `Tooltip` was displayed inside the `MenuButton` and never wrapped it, it only worked when hovering, never when focusing.


## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Focus on the quick-action menu items or Page menu items using keyboard. Verify that the tooltips display when focusing.
2. Add a text and focus on the Bold / Italic / Underline in the floating design menu (toolbar). Verify the tooltips display when focusing.
3. Verify that tooltips show correctly when hovering as well.


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #10884
